docs/source/release_notes/release-notes.rst

```diff
@@ -52,6 +52,12 @@ develop
            Atlas users should use ``AtlasDbMetrics.setMetricRegistry`` to set a ``MetricRegistry``.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1429>`__)
 
+    *    - |new|
+         - There is now a ``TimestampMigrationService`` with the ``fast-forward`` method that can be used to migrate between timestamp services.
+           You will simply need to fast-forward the new timestamp service using the latest timestamp from the old service.
+           This can be done using the :ref:`timestamp forward cli <offline-clis>` when your AtlasDB services are offline.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/?>`__)
```

Comment:
Agreed. We should announce timelock server as one note, and mention that it's in controlled beta state (even if that release note ends up with a long list of PRs).

Other fixes arising from PRs such as this one could be notable, if they can safely be used by products/customers immediately.